### PR TITLE
docs(swagger): update swagger docs

### PR DESCRIPTION
**Pull request by aleem1314**
## Description

Closes: #543 

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -195,6 +195,7 @@ update-swagger-docs: statik
 	else \
 		echo "\033[92mSwagger docs are in sync\033[0m";\
 	fi
+	@rm -f ./client/docs/swagger-ui/swagger.yaml
```
Comment (aleem1314) [inline Makefile:198]: I don't think we need to commit the file.

Comment (anilCSE) [reply Makefile:198]: Ah no, I guess that what being served on /swagger/ rest endpoint.

File: scripts/protoc-swagger-gen.sh [hunk 2]
```diff
@@ -30,5 +30,9 @@ wget "https://raw.githubusercontent.com/cosmos/cosmos-sdk/${SDK_VERSION}/client/
 # all the individual swagger files need to be configured in `config.json` for merging
 swagger-combine ./client/docs/config.json -o ./tmp-swagger-gen/swagger.yaml -f yaml --continueOnConflictingPaths true --includeDefinitions true
 
+# move generated swagger file to swagger-ui directory
+rm -f ./client/docs/swagger-ui/swagger.yaml
```
Comment (anilCSE) [inline scripts/protoc-swagger-gen.sh:34]: is this still valid or do we need to keep swagger.yaml?
```suggestion
```

Comment (aleem1314) [reply scripts/protoc-swagger-gen.sh:34]: Yes, we need this file for `make update-swagger-docs`.

Comment (atheeshp) [reply scripts/protoc-swagger-gen.sh]: we don't need this line, since we are replacing the swagger file in next command.

**Review by amaurym — APPROVED** [review]

**Pull request merged**